### DISCUSSION TITLE
[Feature] Preload contacts in face-recognition name list

### DISCUSSION THREAD

**Discussion by aero31aero** [immich-app/immich#8652] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Either some kind of API import, or just a text field where you can enter some CSV file with a list of your contacts, so that when you're tagging/recognizing people in the `/people` page, you have this data to auto-complete from.

This will make spelling errors in names a thing of the past, and in general improve the UX of face tagging.

I propose providing two options:

1. A text field where a user can manually paste comma separated contact names like: `Alice, Bob, Eve`
2. A text field where a user can specify a url like: `https://example.com/mycontacts` that returns a file formatted like above. Immich can then periodically fetch and update the local contacts list.

I don't want to propose directly hooking up to Google Contacts or implementing Webdav but I feel there should be some configurable way to let Immich know names of people I know in bulk.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#2629
